_stbt/ocr.py

```diff
@@ -250,7 +250,7 @@ def ocr(frame=None, region=Region.ALL,
     if isinstance(tesseract_user_patterns, (bytes, str)):
         tesseract_user_patterns = [tesseract_user_patterns]
 
-    imglog = ImageLogger("ocr")
+    imglog = ImageLogger("ocr", result=None)
```

Comment:
Would doing `imglog.set(result=text)` make sense in this instance?

Reply:
The whole debug template is a bit ad-hoc, it isn't super consistent. The text goes in `output` (including the hocr text from `match_text`), and `result` is only used to draw a region.